compiler/crates/relay-compiler/tests/relay_compiler_integration/fixtures/preloadable_query_typescript.input

```diff
@@ -11,7 +11,8 @@ graphql`
   "eagerEsModules": true,
   "persistConfig": { 
     "file": "./operations.json" 
-  }
+  },
+  "typescriptCheckGeneratedFiles": true
```

Comment:
I don't see any generated files changing as a result of this. Maybe the fixtures are out of date?

Reply:
Updated, but @captbaritone This begs the question, how much of the tests should have this flag on? Or should we leave it off in the tests for the time being?

Reply:
Good question. Maybe just a add a single new test which turns it on. If these integration tests got typechecked (that wold be cool! Especially if/when we pull the library types into this repo!) then I'd stay turn it on for all of them, but without that, all we need to assert is that if you turn on the flag the suppression gets removed, and that's a separate assertion from any pre-existing test, so we can have that as its own test file.

Let me know if you need help figuring out how to add new fixtures.

Reply:
Also, I agree that it would be good to start TS typechecking as a validation step within this repo. That way we can prevent regressions to the generated types. As it stands currently we have to check the output in our own repo first.

Reply:
Yeah, for Flow we have our own codebase as a massive test case. For TypeScript we'll need something to echo that setup. Running typescript on integration tests could be a great way to achieve that pretty cheaply.

We have rust utilities that materialize these fixture files into multiple files in a directory and then run the compiler on them. Maybe that test could also run the TypeScript type check on them? Maybe a little brittle to shell out to TypeScript from within the Rust tests, but probably worth it.

Will require a little fiddling to get it to work both internally and in GitHub CI and ensure we keep the same version(s?) of TypeScript, but I bet we can figure it out.

Here's where we run the integration tests: https://github.com/facebook/relay/blob/main/compiler/crates/relay-compiler/tests/relay_compiler_integration.rs

cc @rbalicki2 regarding upstreaming TypeScript types and how we might add tests which assert their correctness.

Reply:
Added fixture test